Makefile

```diff
@@ -296,7 +296,7 @@ setup-taplo: ## Install taplo TOML formatter
 	@if taplo --version 2>/dev/null | grep -q ${TAPLO_CLI_VERSION}; then \
 		echo "taplo ${TAPLO_CLI_VERSION} already installed"; \
 	else \
-		cargo install taplo-cli --version ${TAPLO_CLI_VERSION} --force; \
+		cargo +nightly install taplo-cli --version ${TAPLO_CLI_VERSION} --force; \
```

Comment:
I think this is fine given the pattern that we have. But we should really give people the option to get dependencies in whatever way works best for them. 

For instance, if I'm using `cargo install`, I'm going to do `cargo +stable install` because I usually have the lastest cargo version no matter the version that this workspace depends on. 

However, I probably wouldn't use `cargo install`, but instead use `cargo-binstall` which is a lot faster than compiling from source.

Reply:
So what I would love to see us do in the future is check for dependencies in the Makefile and print an error if the dependency isn't there.